tests/unit-fail/pta-field-2.ref

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
I am looking at this program and it doesn't look correct to me that it generates an out-of-bounds error. The line of code in question is taking a pointer to an int* and subtracting 1 from it. But the memory being pointed to is the second int* in a struct. So unless I am reading this wrong, it should actually be correctly pointing to the first member of the struct. Can you explain why you think this program should be undefined?